### ISSUE TITLE
中文指标一直上不去 metric is allways bad in Chinese

### ISSUE THREAD

**Issue by BEILOP** [xinyadu/eeqa#12]
我只使用了一条数据如下（Only 1 example to train dev and test）：
{"sentence": ["网上车市从布鲁克斯自动化公司官方获悉，布鲁克斯自动化公司全新蓝河正式发布。"], "s_start": 0, "ner": [[[5, 14, "CompanyName"], [19, 28, "CompanyName"], [30, 32, "Product"]]], "event": [[[32, 36, "NewService.正式发布"], [5, 14, "CompanyName"], [19, 28, "CompanyName"], [30, 32, "Product"]]]}

test结果如下（Test Result）：
[["NewService.正式发布_Product", [30, 32]]]

参数如下（args）：
batch_size = 4
epoch = 0
f1_c = 50.0
f1_i = 50.0
global_step = 36
learning_rate = 1e-06
prec_c = 100.0
prec_i = 100.0
recall_c = 33.333333333333336
recall_i = 33.333333333333336

**Comment by kakuibeyond** [comment]
可能是量太少了吧，毕竟一共就三个待预测的标签
